book/06-github/sections/1-setting-up-account.asc

```diff
@@ -38,7 +38,7 @@ Then select the ``SSH keys'' section along the left-hand side.
 .The ``SSH keys'' link.
 image::images/ssh-keys.png[The ``SSH keys'' link.]
 
-From there, click the "`Add an SSH key`" button, give your key a name, paste the contents of your `~/.ssh/id_rsa.pub` (or whatever you named it) public-key file into the text area, and click ``Add key''.
+From there, click the "`Add an SSH key`" button, give your key a name, paste the contents of your `~/.ssh/id_rsa.pub` (or whatever you named it) public-key file into the text area, and click ``Add key.''
```

Comment:
I have done that in situations where the period could have ambiguously been
confused with content (like for documenting regular expressions) but I
doubt there are very many humans that are going to get confused when the
button says "Add key" and not "Add key." but if you are worried about it,
then just change it to:

...and click the ''Add key'' button.


On Fri, Aug 5, 2016 at 4:03 PM, Ben Straub <notifications@github.com> wrote:

> In book/06-github/sections/1-setting-up-account.asc
> <https://github.com/progit/progit2/pull/630#discussion_r73765996>:
>
> > @@ -38,7 +38,7 @@ Then select the ``SSH keys'' section along the left-hand side.
> >  .The ``SSH keys'' link.
> >  image::images/ssh-keys.png[The ``SSH keys'' link.]
> >
> > -From there, click the "`Add an SSH key`" button, give your key a name, paste the contents of your `~/.ssh/id_rsa.pub` (or whatever you named it) public-key file into the text area, and click ``Add key''.
> > +From there, click the "`Add an SSH key`" button, give your key a name, paste the contents of your `~/.ssh/id_rsa.pub` (or whatever you named it) public-key file into the text area, and click ``Add key.''
>
> This is an example that always seemed ambiguous to me. We're telling the
> user to look for a specific button that reads:
>
> Add key
>
> …without punctuation. They're not looking for a button with a period. So
> when we quote something like that, it seems better to put the exact content
> of the button inside the quotes, just like you would with monospace - the
> period doesn't go inside the single ticks.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/progit/progit2/pull/630/files/2cb2f4600180163db5ac70f1f440e28cbf88c975#r73765996>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABmTbg-iXu6Ht1YJpMCiA-FxDftvbh98ks5qc7M0gaJpZM4JYCxK>
> .
>